modules/ROOT/pages/managing-your-account.adoc

```diff
@@ -186,6 +183,42 @@ include::partial$include-reauthentication-conditions.adoc[]
 
 Connected apps and clients are exempt from reauthentication. If you have a script that makes an API call to change your email address, it might need to accommodate reauthentication.
 
+=== Verify Your Email Address
+
+If you have not verified your email address that you use to log in to Anypoint Platform, a prompt appears to verify your email when you log in.
+You can verify your email using the login prompt, or you can verify your email in your user profile.
+If you are an SSO user, you cannot verify your email address through Anypoint Platform.
```

Comment:
to make it clear, the prompt won't appear for SSO users.

Reply:
```suggestion
SSO users cannot verify their email addresses through Anypoint Platform and do not receive verification prompts.
```